helm/templates/backend-configmap.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "highlight.fullname" . }}-backend
+  labels:
+    app.kubernetes.io/component: backend
+    app.kubernetes.io/part-of: {{ include "highlight.name" . }}
+    {{- include "highlight.labels" . | nindent 4 }}
+data:
+  CLICKHOUSE_ADDRESS: {{ include "highlight.fullname" . }}-clickhouse:9000
+  CLICKHOUSE_DATABASE: default
+  CLICKHOUSE_PASSWORD: ""
+  CLICKHOUSE_USERNAME: default
+  DOPPLER_CONFIG: docker
+  ENVIRONMENT: dev
+  FRONTEND_URI: {{ .Values.frontend.config.frontendUri | quote }}
+  INFLUXDB_BUCKET: dev-bucket
+  INFLUXDB_ORG: dev-org
+  INFLUXDB_SERVER: {{ include "highlight.fullname" . }}-influxdb:8086
+  INFLUXDB_TOKEN: not-a-secure-token
+  KAFKA_SERVERS: {{ include "highlight.fullname" . }}-kafka:{{ .Values.kafka.config.port }}
+  KAFKA_TOPIC: dev
+  OPENSEARCH_INDEX_PREFIX: docker
+  OPENSEARCH_DOMAIN: http://{{ include "highlight.fullname" . }}-opensearch:9200
+  OPENSEARCH_DOMAIN_READ: http://{{ include "highlight.fullname" . }}-opensearch:9200
+  OPENSEARCH_PASSWORD: admin
+  OPENSEARCH_USERNAME: admin
+  OTLP_ENDPOINT: {{ include "highlight.fullname" . }}-collector:4319
+  TZ: America/Los_Angeles
+  PSQL_DB: postgres
+  PSQL_HOST: {{ include "highlight.fullname" . }}-postgres
+  PSQL_PASSWORD: ""
+  PSQL_PORT: "5432"
+  PSQL_USER: postgres
+  PRIVATE_GRAPH_URI: {{ .Values.frontend.config.privateGraphUri | quote }}
+  PUBLIC_GRAPH_URI: {{ .Values.frontend.config.publicGraphUri | quote }}
+  REACT_APP_FRONTEND_ORG: "1"
+  REACT_APP_FRONTEND_URI: {{ .Values.frontend.config.frontendUri | quote }}
+  REACT_APP_IN_DOCKER: "true"
+  REACT_APP_PRIVATE_GRAPH_URI: {{ .Values.frontend.config.privateGraphUri | quote }}
+  REACT_APP_PUBLIC_GRAPH_URI: {{ .Values.frontend.config.publicGraphUri | quote }}
+  REDIS_EVENTS_STAGING_ENDPOINT: {{ include "highlight.fullname" . }}-redis:6379
+  RENDER_PREVIEW: "true"
+  SESSION_FILE_PATH_PREFIX: /tmp/
+  REACT_APP_AUTH_MODE: simple
+  IN_DOCKER: "true"
+  IN_DOCKER_GO: "true"
+  ENABLE_OBJECT_STORAGE: "true"
+  OBJECT_STORAGE_FS: "true"
```

Comment:
Add ON_PREM: "true" to resolve issue with opensearch setup init container

Reply:
Adding this triggers `projectpath.GetRoot()` to change the directory  to the `build` directory and allowing migrations to occur.

Reply:
Great observation and great work at fixing this issue at hand.  
As soon as #6538 and #6597 are merged, the requirements to complete the next major two steps should be resolved.

I'll use this information locally to focus on the remaining checkpoints later this day, as it's midnight in my time zone. Thank you once again for your great assistance in this PR. 🥇

Reply:
Not sure if you have been working consistently with dev flag enabled but might help to flip it to production. I've noticed that when this is flipped we need to run migration either manually or via an init container. 

This becomes more of a problem when you realize the the `migrate-db` service can not actually be run unless you have already initialized all the tables once before. It looks like the runtime / service flags only get checked after all the initial services get spun up and started (ie: oauth server, directly dependent on tables existing in postgres), which becomes a blocker for running the migration service. 

This could potentially require a minor rewrite to elevate the service check above these startups, if a service like `migrate-db` is being run and not the main service there is no sense in initializing all these unused clients just to kill them afterwards.